apache/zeppelin
# [ZEPPELIN-2086] Move zeppelin-web related test files into src/ for efficiency

### What is this PR for?

Since we are using a separated test directory, 

- it's uncomfortable to create test
- it's hard to figure which source is related with this test file.

```javascript
// we have to import like this...
import TableData from '../../../src/app/tabledata/tabledata.js';
```

### What type of PR is it?
[Improvement]

### Todos
* [x] - Fix karma conf
* [x] - Move test files


### What is the Jira issue?

[ZEPPELIN-2086](https://issues.apache.org/jira/browse/ZEPPELIN-2086)

### How should this be tested?

- Execute `npm runt test`
- You should see this message like before

```
webpack: Compiled successfully.
09 02 2017 12:36:01.876:INFO [karma]: Karma v1.3.0 server started at http://localhost:9002/
09 02 2017 12:36:01.878:INFO [launcher]: Launching browser PhantomJS with unlimited concurrency
09 02 2017 12:36:01.888:INFO [launcher]: Starting browser PhantomJS
09 02 2017 12:36:02.372:INFO [PhantomJS 2.1.1 (Mac OS X 0.0.0)]: Connected on socket /#BkEJkZGwxoc6HMA7AAAA with id 42992167
PhantomJS 2.1.1 (Mac OS X 0.0.0): Executed 37 of 70 SUCCESS (0 secs / 0.236 secs)
LOG: function TableData(columns, rows, comment) { ... }
LOG: function TableData(columns, rows, comment) { ... }
PhantomJS 2.1.1 (Mac OS X 0.0.0): Executed 70 of 70 SUCCESS (0.26 secs / 0.352 secs)
```

### Screenshots (if appropriate)

NONE

### Questions:
* Does the licenses files need update? - NO
* Is there breaking changes for older versions? - NO
* Does this needs documentation? - NO
